### PR TITLE
currentRemoteDescription.sdp need not match remoteDescription.sdp

### PR DESCRIPTION
**Pull request by aboba**
Fix for Issue https://github.com/w3c/webrtc-pc/issues/921

### CODE REVIEW

File: webrtc.html [hunk 1]
```diff
@@ -1112,7 +1112,9 @@
                 that have been supplied via <code><a data-for=
                 "RTCPeerConnection">addIceCandidate()</a></code> since the
                 offer or answer was created. Prior to being set, it returns
-                null.</p>
+                null.  Note that the value of <code>currentRemoteDescription.sdp</code>
+                need not match the value of
+                <code>setRemoteDescription().sdp</code>.</p>
```
Comment (stefhak) [inline webrtc.html]: do you mean ```remoteDescription.sdp``` ?

Comment (aboba) [inline webrtc.html]: Yes, that is what Fippo referred to in the Issue.

**Comment by alvestrand** [comment]
We need to say the same thing about localDescription

**Comment by aboba** [comment]
@fippo  Does this work for you?

**Comment by fippo** [comment]
LGTM, thanks!

**Pull request merged**